src/main/java/christmas/domain/user/VisitDate.java

```diff
@@ -0,0 +1,52 @@
+package christmas.domain.user;
+
+import christmas.constant.ErrorMessage;
+import christmas.util.StringConvertor;
+import java.time.DayOfWeek;
+import java.time.LocalDate;
+import java.util.List;
+
+public class VisitDate {
+    private static final int START_DATE = 1;
+    private static final int END_DATE = 31;
+    private static final int CHRISTMAS_D_DAY = 25;
+    private static final List<Integer> SPECIAL_DAYS = List.of(3, 10, 17, 24, 25, 31);
+    private final LocalDate date;
+
+    private VisitDate(LocalDate date) {
+        this.date = date;
+    }
+
+    public static VisitDate from(String input) {
+        int day = StringConvertor.convertStringToInt(input);
+        validateRange(day);
+        LocalDate visitDate = LocalDate.of(2023, 12, day);
+        return new VisitDate(visitDate);
+    }
+
+    private static void validateRange(int date) {
+        if (date < START_DATE || date > END_DATE) {
+            throw new IllegalArgumentException(ErrorMessage.VISIT_DATE_ERROR);
+        }
+    }
+
+    public boolean isWeekday() {
+        DayOfWeek dayOfWeek = date.getDayOfWeek();
+        return dayOfWeek != DayOfWeek.FRIDAY && dayOfWeek != DayOfWeek.SATURDAY;
+    }
+
+    public boolean isSpecialDay() {
+        return SPECIAL_DAYS.contains(date.getDayOfMonth());
+    }
+
+    public boolean isChristmasDday() {
+        if (date.getDayOfMonth() <= CHRISTMAS_D_DAY) {
+            return true;
+        }
+        return false;
+    }
```

Comment:
```suggestion
    public boolean isChristmasDday() {
        return date.getDayOfMonth() <= CHRISTMAS_D_DAY;
    }
```

true 라서 true 를 리턴하고, false 라서 false 를 리턴하는 것은 자명해서, 이렇게 표현해보면 어떨까요?

Reply:
처음 조건문을 배웠을 때 다른 사람들의 코드에서 해로님 처럼 간결하게 표현한 것에 대해 "언제 ture고, 언제 false지?"
하는 혼동이 있어서 그 때부터 간단하게 표현할 수 있는 것도 if문을 감싸서 반환 시키던 습관이 있었습니다.ㅋㅋㅋ 🤣
이제는 해로님이 제시한 방법처럼 간결하게 표현할 수 있는 조건문은 간결하게 표현할 수 있도록 바꿔봐야겠습니다!